File: bench/elixir/fib.ex
defmodule Bfib do
  def fib(n) when n < 2, do: n
  def fib(n), do: fib(n - 1) + fib(n - 2)

  def main do
    n = String.to_integer(System.get_env("BENCH_N") || "35")
    IO.puts(fib(n))
  end
end

File: bench/erlang/fib.erl
#!/usr/bin/env escript
%%! -mode(compile)
main(_) -> io:format("~w~n", [fib(bench_n(35))]).
bench_n(D) -> case os:getenv("BENCH_N") of false -> D; V -> list_to_integer(V) end.
fib(N) when N < 2 -> N;
fib(N) -> fib(N-1) + fib(N-2).
